### DISCUSSION TITLE
Dominio de las complejidades en la pregunta 2.

### DISCUSSION THREAD

**Discussion by eayaach** [IIC2133-PUC/2024-2#105] [Tarea 1] [answered]
Consulta el k que hace referencia al dominio en la pregunta 2. Este k se refiere a la cantidad de dígitos de los numeros a usar o al rango, tipo si age esta entre 0  y 99 el dominio seria 100 o 2?

**Comment by Segismundios** [comment, marked as answer]
Hola @eayaach. El dominio se refiere al rango de valores posibles. 
Saludos!
